FogCreek/Glitch-Community
# Pickle suggestion: small style fixes

## Links
* https://pickle-suggestion.glitch.me
* https://app.zenhub.com/workspaces/community-5d07d1359dee6059a9688176/issues/fogcreek/glitch-community-work/254
* https://app.zenhub.com/workspaces/community-5d07d1359dee6059a9688176/issues/fogcreek/glitch-community-work/253
* https://app.zenhub.com/workspaces/community-5d07d1359dee6059a9688176/issues/fogcreek/glitch-community-work/241

## GIF/Screenshots:
Added spacing to collection page:
![Screen Shot 2019-08-06 at 2 33 32 PM](https://user-images.githubusercontent.com/6620164/62566550-36a59e80-b857-11e9-92bb-e2a51ec9a085.png)

Before and After spacing of privacy toggle for super users:
![Screen Shot 2019-08-06 at 2 31 46 PM](https://user-images.githubusercontent.com/6620164/62566456-ffcf8880-b856-11e9-8b13-1437922bfeea.png)

Fixed issue where it was hard to click on small items that had a tooltip, a little hard to illustrate this one in gif form, but hopefully it at least points you to the areas to test with between prod and the remix. Essentially we're having issues selecting things like the privacy button and team avatars because the tooltip triangle is taking up too much height:
![](http://g.recordit.co/CTJS8YRL0n.gif)
![](http://g.recordit.co/wS9HSPtOYx.gif)

## Changes:
* adds the default margin to our collection help text
* adds a bit more space to our privacy toggle to ensure we never overlap with user avatars (this bug I think only affected super users, because when you're authed you already get enough padding)
* changes the height to the triangle bit of the tooltip so that we're still able to click right above the tooltip. Interestingly if I remove it on chrome that's solves the issue, but on firefox it looks like the default is 100% unless you specify it. If I set it to 1% that solves the issue in firefox and in chrome. 🤷‍♀ feel free to skool me in this, I'm just finagling stuff and it seems fixed. Checked this solution safari and it seemed ok to me as well. 

## How To Test:
* check out a collection page, make sure padding looks good on the help text below a collection
* mark a project as private on a test account, make yourself a super user and view that project from your superuser account, make sure you're able to click on user avatars and that the tooltip is not too burdensome.
* try toggling privacy badges, make sure you can still select the tooltip text, but that you also can click the button, check tooltips throughout the site to ensure I'm not missing anything  

## Feedback I'm looking for:
- this all look ok for you? any unintended consequences of these choices I'm not thinking about?
